sound/soc/sof/ipc4-topology.c

```diff
@@ -276,6 +276,20 @@ static int sof_ipc4_get_audio_fmt(struct snd_soc_component *scomp,
 
 static void sof_ipc4_widget_free_comp(struct snd_sof_widget *swidget)
 {
+
+	switch (swidget->id) {
+	case snd_soc_dapm_dai_in:
+	case snd_soc_dapm_dai_out:
+	{
```

Comment:
is this correct @bardliao? the patch Add support for parsing AIF_IN/AIF_O... doesnt addsupport for DAI comps yet

Reply:
> is this correct @bardliao? the patch Add support for parsing AIF_IN/AIF_O... doesnt addsupport for DAI comps yet

Thanks, updated.